packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.DragDrop.Example.tsx

```diff
@@ -159,22 +164,24 @@ export class DetailsListDragDropExample extends React.Component<
     };
   }
 
-  private _onItemInvoked(item: any): void {
+  private _onItemInvoked = (item: IExampleItem): void => {
     alert(`Item invoked: ${item.name}`);
-  }
+  };
 
-  private _onRenderItemColumn(item: any, index: number, column: IColumn): JSX.Element {
+  private _onRenderItemColumn = (item: IExampleItem, index: number, column: IColumn): JSX.Element | string => {
     if (column.key === 'name') {
       return <Link data-selection-invoke={true}>{item[column.key]}</Link>;
     }
 
-    return item[column.key];
-  }
+    return (item as any)[column.key];
```

Comment:
Any reason why? (might be a good idea to add a comment here)

Reply:
The interface doesn't have an index signature, but in context of the example the key will always be a property name from the item.